examples/slurm_allreduce.ipynb

```diff
@@ -2,87 +2,56 @@
-      "slurm.utils 2025-08-29 21:02:45 INFO \n",
-      "===== Server Info =====\n",
-      "{\n",
-      "  \"name\": \"418\",\n",
-      "  \"server_handle\": \"slurm:///418\",\n",
-      "  \"state\": \"RUNNING\",\n",
-      "  \"meshes\": {\n",
-      "    \"mesh0\": {\n",
-      "      \"host_type\": \"__UNSET__\",\n",
-      "      \"hosts\": 2,\n",
-      "      \"gpus\": -1,\n",
-      "      \"hostnames\": [\n",
-      "        \"gpu-queue-st-gpu-compute-1\",\n",
-      "        \"gpu-queue-st-gpu-compute-2\"\n",
-      "      ]\n",
-      "    }\n",
-      "  }\n",
-      "}\n",
-      "__main__ 2025-08-29 21:02:45 INFO computing world size...\n",
-      "monarch._src.actor.allocator 2025-08-29 21:02:45 INFO no match label `procmesh.monarch.meta.com/name` specified in alloc constraints\n",
-      "monarch._src.actor.allocator 2025-08-29 21:02:45 INFO found a single proc mesh `mesh0` in slurm:///418, will allocate on it\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO no AF_INET6 address that can bind TCP sockets for `gpu-queue-st-gpu-compute-1:26600` (error: [Errno -5] No address associated with hostname)\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO resolved AF_INET address `10.0.2.236:26600` for `gpu-queue-st-gpu-compute-1:26600`\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO no AF_INET6 address that can bind TCP sockets for `gpu-queue-st-gpu-compute-2:26600` (error: [Errno -5] No address associated with hostname)\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO resolved AF_INET address `10.0.2.132:26600` for `gpu-queue-st-gpu-compute-2:26600`\n",
-      "monarch._src.actor.allocator 2025-08-29 21:02:45 INFO initializing alloc on remote allocator addresses: ['tcp!10.0.2.236:26600', 'tcp!10.0.2.132:26600']\n",
-      "monarch._src.actor.allocator 2025-08-29 21:02:45 INFO no match label `procmesh.monarch.meta.com/name` specified in alloc constraints\n",
-      "monarch._src.actor.allocator 2025-08-29 21:02:45 INFO found a single proc mesh `mesh0` in slurm:///418, will allocate on it\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO no AF_INET6 address that can bind TCP sockets for `gpu-queue-st-gpu-compute-1:26600` (error: [Errno -5] No address associated with hostname)\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO resolved AF_INET address `10.0.2.236:26600` for `gpu-queue-st-gpu-compute-1:26600`\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO no AF_INET6 address that can bind TCP sockets for `gpu-queue-st-gpu-compute-2:26600` (error: [Errno -5] No address associated with hostname)\n",
-      "monarch.tools.network 2025-08-29 21:02:45 INFO resolved AF_INET address `10.0.2.132:26600` for `gpu-queue-st-gpu-compute-2:26600`\n",
-      "monarch._src.actor.allocator 2025-08-29 21:02:45 INFO initializing alloc on remote allocator addresses: ['tcp!10.0.2.236:26600', 'tcp!10.0.2.132:26600']\n"
+      "Found cached job at path: .monarch/job_state.pkl\n",
```

Comment:
We can probably disable cached job for these examples (pass as none)

Reply:
Seconded, please remove all outputs in examples. Keep it to just the code to run